README.md

```diff
@@ -27,63 +47,99 @@ You will need to perform the following on your development machine:
+| ✅            | `start`             | Runs complete test and build suite and starts the dev server.                                 |
+| ✅            | `serve`             | Runs build suite and starts the dev server.                                                   |
+| ✅            | `build`             | Runs build suite and outputs the result into `dist`. Takes optional `base` value as argument. |
+| ✅            | `test`              | Runs the test suite with eslint, and stylelint.                                               |
+| ✅            | `test:eslint`       | Runs only eslint.                                                                             |
+| ✅            | `test:stylelint`    | Runs only stylelint.                                                                          |
+| ✅            | `test:mocha`        | Runs only the test suite.                                                                     |
+| ❌            | `deploy`            | Runs `build` script, then pushes the output to GH Pages.                                      |
+| ❌            | `deploy:clean`      | Runs `build` script, clears `gh-pages` branch, then pushes the output to GH Pages.            |
+| ❌            | `deploy:folder`     | Runs `build` script, then pushes the output to the specified folder in GH Pages.              |
+| ❌            | `deploy:staging`    | Initiates procedures for deploying to staging. (Package specific)                             |
+| ❌            | `deploy:production` | Initiates procedures for deploying to production. (Package specific)                          |
+
+**Please follow the README of each package you intend to work with on how to get set up and their custom scripts.** However, the above scripts can be run from the root directory in the following manner.
+
+### Package names
+Each package is named with the `deriv-` prefix, however for the scripts above, you do not need to add the `deriv-` prefix as the scripts already prefix the 1st argument of the script with `deriv-`.
+
+You can find the names of packages by first navigating to the `packages` folder. Each subfolder is a package, and contains a `package.json` file. The value of the `name` key in `package.json` is the package name.
 
 ### Usage
+#### Starting a Development Server
+
+If you wish to work on Core, simply run `npm run serve core`.
+
+But for working on any of the other packages (such as Trader, Bot, P2P), perform the following:
+1. Open 2 terminals.
+2. Run `npm run serve {package name}` in the first one. e.g.: `npm run serve translations`, `npm run serve bot`, etc.
+3. Then run `npm run serve core` in the second one.
+
 #### How to Clean Packages
-If you intend to remove `node_modules` folder(s) from the projects, please run `lerna clean` and follow the instructions.
+If you intend to remove `node_modules` folder(s) from the projects, please run `npm run clean` from the root of the project.
```

Comment:
are we moving away from lerna?

Reply:
@reza-fs We are not. These `npm` commands that are from the root of the project (and only the root of the project) are simply abstractions over the `lerna` commands for developer ease-of-use. You can always use lerna commands and if your needs surpass the limited options provided in the root package.json, you definitely should just use lerna according to [lerna docs](https://github.com/lerna/lerna/).